### ISSUE TITLE
[opencv4] Feature python tries to install into system directory

### ISSUE THREAD

**Issue by Neumann-A** [microsoft/vcpkg#35060]
### Operating system

Windows

### Compiler

MSVC

### Steps to reproduce the behavior

```Shell
(Maybe have anaconda installed as system python).

opencv4[python] tries to install python stuff into ProgramData/anaconda/...
```

### Failure logs

Trace log showed __INSTALL_PATH_PYTHON3 set to that path in:
`E:/all/vcpkg/buildtrees/opencv4/src/4.8.0-28394c3990.clean/modules/python/common.cmake(197):  set(__INSTALL_PATH_PYTHON3 C:/ProgramData/anaconda3/Lib/site-packages/cv2/python-3.11 CACHE INTERNAL  )`

**Comment by Cheney-W** [comment]
Stay active

**Comment by Neumann-A** [comment]
@cenit has this been fixed?

**Comment by cenit** [comment]
yes at least for me now python is installed properly (since 4.9). Please double check if you can

**Comment by cenit** [comment]
i missed this issue when browsing them for "fixed" auto resolution tag in the PR
